SEImplementation/src/lib/Configuration/WeightImageConfig.cpp

```diff
@@ -129,6 +129,10 @@ void WeightImageConfig::initialize(const UserValues& args) {
   if (args.count(WEIGHT_THRESHOLD) != 0) {
     auto threshold = args.find(WEIGHT_THRESHOLD)->second.as<double>();
     m_weight_threshold = computeWeightThreshold(m_weight_type, threshold);
+    if (flux_scale != 1. && m_absolute_weight){
+	// adjust the m_weight_threshold
+	m_weight_threshold *= flux_scale;
```

Comment:
Wait... why is this multiplied by `flux_scale` and but the weight_image is multiplied by `flux_scale * flux_scale`?

Reply:
You are right. Sorry for this.

Shall I fix it?

Reply:
Sure but which one is correct? I'm no longer sure how it works.

Reply:
`flux_scale*flux_scale` is the correct one. The detection image is multiplied by `flux_scale`. The weight image, which is a variance image, is multiplied by  `flux_scale*flux_scale` to not affect the signal-to-noise ratio. The threshold must follow the variance/weight image it refers to